xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
+    This target can be used to store your backups. To create a Rsync server,
+    do the following:
    </para>
-<screen>rsync -baz -e ssh<!--
-   --> /home/&exampleuser_plain;/ &exampleuser_plain;@&exampleserver;:backup</screen>
-   <para>
-    The following command is used to play the directory back:
-   </para>
-<screen>rsync -az -e ssh<!--
-   --> &exampleuser_plain;@&exampleserver;:backup /home/&exampleuser_plain;/</screen>
-   <para>
-    Up to this point, the handling does not differ much from that of a regular
-    copying tool, like scp.
-   </para>
-   <para>
-    rsync should be operated in <quote>rsync</quote> mode to make all its
-    features fully available. This is done by starting the rsyncd daemon on one
-    of the systems. Configure it in the file
-    <filename>/etc/rsyncd.conf</filename>. For example, to make the directory
-    <filename>/srv/ftp</filename> available with rsync, use the following
-    configuration:
-   </para>
-<screen>gid = nobody
-uid = nobody
-read only = true
-use chroot = no
-transfer logging = true
-log format = %h %o %f %l %b
-log file = /var/log/rsyncd.log
-
-[FTP]
-        path = /srv/ftp
-        comment = An Example</screen>
-   <para>
-    Then start rsyncd with <command>systemctl start rsyncd</command>. rsyncd
-    can also be started automatically during the boot process. Set this up by
-    activating this service in the &yast; <guimenu>Services Manager</guimenu>
-    or by manually entering the command:
-   </para>
-<screen>&prompt.root; systemctl enable rsyncd</screen>
-   <para>
-<!--<remark>taroth 2014-02-25: systemd - what about xinetd? is this
+   <procedure>
+    <title>Setting Up the Rsync Server</title>
+    <step>
+     <para>
+      On &wsI;, create a directory to store all your backup files.
+      In this example, we use <filename>/var/backup</filename>:
+     </para>
+     <screen>&prompt.root;<command>mkdir</command> /var/backup</screen>
+    </step>
+    <step>
+     <para>Decide about the ownership. In this case, the directory is owned by
+      user &exampleuser; in group <systemitem class="groupname"
+       >users</systemitem>:</para>
+     <screen>&prompt.root;<command>chown</command> &exampleuser_plain;.users /var/backup</screen>
+    </step>
+    <step>
+     <para>Configure the rsyncd daemon.</para>
+     <para>We will separate the configuration file into a main file and some
+       <quote>modules</quote> which holds your backup target. This makes it
+      easier to add additional targets later. Global values can be stored in
+       <filename>/etc/rsyncd.d/*.inc</filename> files, whereas your modules are
+      placed in <filename>/etc/rsyncd.d/*.conf</filename> files: </para>
+     <substeps>
+      <step>
+       <para>Create a directory <filename>/etc/rsyncd.d/</filename>:</para>
+       <screen>&prompt.root;<command>mkdir</command> /etc/rsyncd.d/</screen>
+      </step>
+      <step>
+       <para>
+        In the main configuration file
```

Comment:
In the main configuration file <filename>/etc/rsyncd.conf</filename>

**In the main configuration file <filename>/etc/rsyncd.conf</filename>,**